docs/admin/node.md

```diff
@@ -96,23 +101,26 @@ the following conditions mean the node is in sane state:
 ]
 ```
 
+If the Status of the Ready condition
+is Unknown or False for more than five minutes, then all of the Pods on the node are terminated.
+
 ### Node Capacity
 
 Describes the resources available on the node: CPUs, memory and the maximum
-number of pods that can be scheduled on this node.
+number of pods that can be scheduled onto the node.
 
 ### Node Info
 
 General information about the node, for instance kernel version, kubernetes version
 (kubelet version, kube-proxy version), docker version (if used), OS name.
-The information is gathered by Kubernetes from the node.
+The information is gathered by Kubelet from the node.
 
 ## Node Management
 
 Unlike [Pods](../user-guide/pods.md) and [Services](../user-guide/services.md), a Node is not inherently
 created by Kubernetes: it is either created from cloud providers like Google Compute Engine,
```

Comment:
I'm not sure I agree with this one -- it is actually created by Kubernetes, based on information it gets from the cloud provider. So I think "from" is correct.